docs/maintainer-guide/governance.md

```diff
@@ -81,6 +89,17 @@ To become a TSC member:
 
 A Committer is invited to become a TSC member by existing TSC members. A nomination will result in discussion and then a decision by the TSC.
 
+#### Process for Adding TSC Members
+
+1. Add the GitHub user to the "ESLint TSC" team
+2. Set the GitHub user to be have the "Owner" role for the ESLint organization
+3. Send welcome email with link to maintainer guide
+4. Add TSC member to the README
+5. Invite to Gitter TSC chatroom
+6. Make TSC member an admin on the ESLint team mailing list
+7. Add TSC member as an admin to ESLint Twitter Account on Tweetdeck
+8. Tweet congratulations to the new TSC member from the ESLint Twitter account
```

Comment:
I say to add them as an admin on tweetdeck, is there some other permission you're referring to?